ideascube/mediacenter/models.py

```diff
@@ -106,7 +106,7 @@ def index_source(self):
 
     @property
     def index_tags(self):
-        return self.tags.slugs()
+        return self.tags.names()
```

Comment:
Why not search on both?

We now lower-case all tags, which means the `slug` will now be the `name` with a few differences:

* punctuation and spaces replaced by `-`
* non-ascii characters replaced by their `unidecode`-ed counterpart (modulo the fact that some languages are poorly handled by our slugifying process, see #118)

As a result, if we index/search on both the `name` and the `slug`, then a search for the term "marche" will return both tags containing "marche" and "marché".

Oh look! That feature we wanted! For free! 😃